includes/definitions/ruckuswireless-unleashed.yaml

```diff
@@ -0,0 +1,26 @@
+os: ruckuswireless-unleashed
+text: 'Ruckus Wireless Unleashed'
+type: wireless
+icon: ruckus
+mib_dir:
+    - ruckus
+over:
+    - { graph: device_bits, text: Traffic }
+    - { graph: device_wireless_ap-count, text: 'Connected APs' }
+    - { graph: device_wireless_clients, text: 'Number of Clients' }
+register_mibs:
```

Comment:
You can remove the register_mibs on the new OS.  It is only on the existing one for legacy reasons.